packages/validation/src/rule-provider.ts

```diff
@@ -620,7 +620,7 @@ const contextualProperties: Readonly<Set<string>> = new Set([
 export class ValidationMessageProvider implements IValidationMessageProvider {
 
   private readonly logger: ILogger;
-  protected registeredMessages: WeakMap<IValidationRule, Map<string|symbol, Interpolation | PrimitiveLiteralExpression>> = new WeakMap();
+  protected registeredMessages: WeakMap<IValidationRule, Map<string | symbol, Interpolation | PrimitiveLiteralExpression>> = new WeakMap();
```

Comment:
OCD trigger ;)
Probably revert this. If this file is not touched by your change otherwise, why format it?